Readme.md

```diff
@@ -10,28 +10,34 @@ Current React Native tested versions from `52.x` but it may work in earliers too
 This module can be use in any project as a dev dependency, but it focus in RN projects.
```

Comment:
Should say can be used for vanilla JS, React (web) or React Native projects.

Reply:
Reworded